README.md

```diff
@@ -4,40 +4,43 @@ To allow users to provide feedback about the ONS website
+| ------------------------------ | --------------------------- | ------------------------------------------------------------------------------------------------------------------ |
+| API_ROUTER_URL                 | <http://localhost:23200/v1> | The URL of the [dp-api-router](https://github.com/ONSdigital/dp-api-router)                                        |
+| BIND_ADDR                      | localhost:25200             | The host and port to bind to                                                                                       |
+| CENSUS_TOPIC_ID                | 4445                        | The census topic id                                                                                                |
+| DEBUG                          | false                       | Enable debug mode                                                                                                  |
+| ENABLE_CENSUS_TOPIC_SUBSECTION | false                       | Enable census topic subsection                                                                                     |
+| ENABLE_NEW_NAVBAR              | false                       | Enable new navigation bar                                                                                          |
+| GRACEFUL_SHUTDOWN_TIMEOUT      | 5s                          | The graceful shutdown timeout in seconds (`time.Duration` format)                                                  |
+| HEALTHCHECK_INTERVAL           | 30s                         | Time between self-healthchecks (`time.Duration` format)                                                            |
+| HEALTHCHECK_CRITICAL_TIMEOUT   | 90s                         | Time to wait until an unhealthy dependent propagates its state to make this app unhealthy (`time.Duration` format) |
+| MAIL_HOST                      | ""                          | The host for the mail server.                                                                                      |
+| MAIL_PORT                      | ""                          | The port for the mail server.                                                                                      |
+| MAIL_USER                      | ""                          | A user on the mail server.                                                                                         |
+| MAIL_PASSWORD                  | ""                          | The password for the mail server user.                                                                             |
+| FEEDBACK_TO                    | ""                          | Receiver email address for feedback.                                                                               |
+| FEEDBACK_FROM                  | ""                          | Sender email address for feedback.                                                                                 |
+| IS_PUBLISHING_MODE             | false                       |
```

Comment:
might be out of scope for this task and can see you're only documenting env vars but in other apps (see: [dp-frontend-search-controller](https://github.com/ONSdigital/dp-frontend-search-controller/blame/develop/config/config.go#L31)) we use `IS_PUBLISHING` do you think it's worth keeping them consistent? happy if you think this is worth looking at at a different time

Reply:
I was doing a bit of tidying up and decided to put these in alpha order, that'll learn me. I'm all for consistency. Will rename here and update `dp-configs`